workshop/exercise-9/README.md

```diff
@@ -159,7 +211,7 @@ And that the generated `ci/release/kabanero-index.yaml` has a section like the f
   pipelines:
   - id: custom-pipeline
     sha256: e3c3050850bf88b97c8fba728592d0cf671bb9d27b582ebaa9f90d939bfa60a5
-    url: https://github.com/stevemar/collections/releases/download/0.2.1/incubator.my-nodejs-express.v0.2.9.pipeline.custom-pipeline.tar.gz
+    url: https://github.com/stevemar/collections/releases/download/0.2.1/incubator.my-nodejs-express.v0.3.0.pipeline.custom-pipeline.tar.gz
 ```
 
 Like we did in Exercise 6, we need to ensure the url for our custom pipeline points at our new collection:
```

Comment:
I don't think we need to do this, do we? (we removed this step in ex6 by setting the TRAVIS_TAG). Maybe we just ask them to look at it, to confirm it does contain the right tag?

Reply:
Oh, this is a section where we're asking the student to check a few files, and not make changes, but in reviewing the comment, it looks like there's an extra copy of the example of what to check so I've removed that.